tests/integration/conftest.py

```diff
@@ -1,23 +1,21 @@
 import pytest
+from django.test import Client
 
 from tests import factories
 
 
-@pytest.fixture
-def client_with_user(client):
-    def _client(session_user):
-        session_user.setdefault("username", "test")
-        user = factories.create_user(**session_user)
-        session = client.session
+class AirlockClient(Client):
+    def login(self, username="testuser", workspaces={}, output_checker=False):
```

Comment:
Seeing mutable default arguments makes me nervous.  Could you change to `workspaces=None`?  `factories.create_user` will then do the right thing.